.changeset/tsconfig-dangerously.md

```diff
@@ -0,0 +1,12 @@
+---
+'sku': minor
+---
+
+`srcPaths` no longer affects `tsconfig.json#include`. Instead, you can use [`dangerouslySetTSConfig`][dangerous] option to have more control over which files are included in the TypeScript compilation process.
+
+Previously, sku managed the `include` field in `tsconfig.json`, but this was problematic for projects that wanted more fine grained control over what is included and/or excluded from compilation.
+
+> **Note**: If you were previously using [`srcPaths`][srcpaths] for this purpose, you should remove the paths which are not source files.
```

Comment:
Maybe this should be turned into more of a migration guide rather than just a note. Though I guess it might be hard to provide a generic migration guide as it won't necessarily be the same for all consumers. 🤔